influx/querybuilder.go

```diff
@@ -275,6 +275,11 @@ func (q Query) Fill(value string) Query {
 	return q
 }
 
+// LastPoint limits the query to return only the last element.
```

Comment:
I think this comment should make it clear that it changes the `OrderByTime` of the query. I mean, if someone executes something like:

```go
influx.NewQuery().On(...).OrderByTime("ASC").LastPoint()
```

It can be a bit surprising.